### PR TITLE
Expected output for transceiver DOM CLI

### PR DESCRIPTION
**Pull request by mihirpat1**
Add the expected output for `show interfaces transceiver dom <port>` CLI for all types of transceivers.

Overall, this CLI will display the real time values and threshold data for the DOM sensors on the particular SFP. The CLI is planned to read data from TRANSCEIVER_DOM_SENSOR and TRANSCEIVER_DOM_THRESHOLD STATE_DB tables.

The current output is classified in 3 sections.
Section 1 will be displayed for all transceivers.
Section 2 will be displayed only if the transceiver is CMIS or C_CMIS based
Section 3 will be displayed only if the transceiver is C_CMIS based

### CODE REVIEW

**Comment by mihirpat1** [comment]
@prgeor @qinchuanares @keboliu @shyam77git  - It will be great if you can help in review this.

**Review by qinchuanares — APPROVED** [review]

**Comment by keboliu** [comment]
In PR https://github.com/sonic-net/sonic-utilities/pull/2630 "**show interfaces transceiver info {interface}**" was added as a warper to "**sfpshow eeprom -p {interface}**", will this command be a similar case? a wraper to "**sfpshow eeprom -p {interface} --dom**"?

**Comment by mihirpat1** [comment]
> In PR [sonic-net/sonic-utilities#2630](https://github.com/sonic-net/sonic-utilities/pull/2630) "**show interfaces transceiver info {interface}**" was added as a warper to "**sfpshow eeprom -p {interface}**", will this command be a similar case? a wraper to "**sfpshow eeprom -p {interface} --dom**"?

@keboliu We plan to deprecate "sfpshow eeprom" in future and use "**show interfaces transceiver info {interface}**" or "**show interfaces transceiver dom {interface}**" for dumping EEPROM info or DOM related information.
Currently, we plan to add "**show interface transceiver dom {port}**" as a wrapper to "**sfpshow dom {interface}**"

File: doc/platform_api/CMIS_and_C-CMIS_support_for_ZR.md [hunk 1]
```diff
@@ -747,6 +747,53 @@ This command displays information for all the interfaces for the transceiver req
...
+    Laser current frequency [GHz]
+    Tx config power [dBm]
+    Case Temperature [C]
+    Voltage [V]
+    Tx Bias Channel [mA]          [Val1, .. Valn]
+    Tx Bias Power [dBm]           [Val1, .. Valn]
+    Rx Power [dBm]                [Val1, .. Valn]
+
+    Section 2 - Fields specific for CMIS + CCMIS
+    Lase Temperatue [C]
```
Comment (prgeor) [inline doc/platform_api/CMIS_and_C-CMIS_support_for_ZR.md:772]: type Lase -> Laser?

**Review by prgeor — APPROVED** [review]
@mihirpat1 approving with minor comment

**Comment by shyam77git** [comment]
@longhuan-cisco to please review this

File: doc/platform_api/CMIS_and_C-CMIS_support_for_ZR.md [hunk 2]
```diff
@@ -747,6 +747,53 @@ This command displays information for all the interfaces for the transceiver req
...
+    CD – Short Link [Ps/nm]
+    CFO [MHz]
+    DGD [Ps]
+    eSNR [dB]
+    OSNR [dB]
+    PDL [dB]
+    Rx signal power[dBm]
+    Rx total power[dBm]
+    SOPMD [Ps^2]
+    Tx Power [dBm]
```
Comment (longhuan-cisco) [inline doc/platform_api/CMIS_and_C-CMIS_support_for_ZR.md:794]: I saw tx_power field is filled in get_transceiver_bulk_status() for all SFF8636 (QSFP28), SFF8436 (QSFP+), SFF8472 (QSFP+). Shall we show tx_power in the common section?

Comment (mihirpat1) [reply doc/platform_api/CMIS_and_C-CMIS_support_for_ZR.md]: I have moved this to the common section now.

**Review by longhuan-cisco — APPROVED** [review]

**Review by prgeor — APPROVED** [review]

**Pull request merged**